packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -1851,6 +1946,52 @@ class MediaQuery extends InheritedModel<_MediaQueryAspect> {
     _MediaQueryAspect.supportsShowingSystemContextMenu,
   )?.supportsShowingSystemContextMenu;
 
+  /// Returns the [MediaQueryData.lineHeightScaleFactorOverride] for the nearest
+  /// [MediaQuery] ancestor or null, if no such ancestor exists.
```

Comment:
Uhh, this also returns null if the override doesn't exist right? It makes sense to me that no MediaQuery == no override, but the documentation might need to be changed a little.